src/screens/runPlan/runPlanList/PlanItemListItem.tsx

```diff
@@ -55,22 +56,18 @@ const styles = StyleSheet.create({
   },
   nameTextColor: {
     color: palette.textBlack,
+      flex: 1,
```

Comment:
should indent be 2 or 4? it's different for different styles

Reply:
2 in general :) Will fix this